collect_app/src/main/java/org/odk/collect/android/activities/CollectAbstractActivity.java

```diff
@@ -70,6 +73,7 @@ public boolean isInstanceStateSaved() {
     @Override
     protected void attachBaseContext(Context base) {
         super.attachBaseContext(base);
+        DaggerUtils.getComponent(base).inject(this);
```

Comment:
Why does this move here?

Reply:
applyOverrideConfiguration() (-> updateConfigurationIfSupported()) which is one line below uses injected `preferencesRepository` so we had to setup dagger earlier.

Reply:
Ah ok makes sense!